CovertInfiltration/Src/CovertInfiltration/Classes/X2Helper_Infiltration.uc

```diff
@@ -539,6 +539,30 @@ static function XComGameState_MissionSite GetMissionStateFromActivity (XComGameS
 	return XComGameState_MissionSite(`XCOMHISTORY.GetGameStateForObjectID(ActivityState.PrimaryObjectRef.ObjectID));
 }
 
+static function XComGameState_HeadquartersProjectTrainAcademy GetTrainAcademyProject(StateObjectReference UnitRef)
```

Comment:
Why was `X2Helper_Infiltration::GetAcademyProjectForUnit` not enough?

Reply:
i had no idea that existed